RadixWallet/Clients/AccountPortfoliosClient/AccountPortfoliosClient+State.swift

```diff
@@ -0,0 +1,285 @@
+import Foundation
+
+// MARK: - Definition
+extension AccountPortfoliosClient {
+	public struct AccountPortfolio: Sendable, Hashable {
+		public var account: OnLedgerEntity.Account
+		public var poolUnitDetails: Loadable<[OnLedgerEntitiesClient.OwnedResourcePoolDetails]> = .idle
```

Comment:
pool and stake details are loadable, so that those are updated dynamically once the loading result is received.